src/AbstractSubscription.js

```diff
@@ -97,7 +97,7 @@ export default class AbstractSubscription extends Subscription {
             try {
                 this.emit('no_resend', response)
```

Comment:
First look at this code I was "wait what... how can emit throw?" but of course it can since it synchronously calls the handlers. This is bad if code is meant for library use. Consider:
```
const EE = require("events")

const ee = new EE()

ee.on("error", console.error)

ee.on("asdf", console.log)
ee.on("asdf", console.log)
ee.on("asdf", x => {
    throw new Error(x)
})
ee.on("asdf", console.log)
ee.on("asdf", console.log)

ee.emit("asdf", "testing lol")
```
prints
```
testing lol
testing lol
/Users/jtakalai/temp/eventEmitterTest.js:10
    throw new Error(x)
    ^

Error: testing lol
```
Meaning the order of handlers causes some be called and some not if a handler throws in the middle. That's just not cool. For our code, we can control that, but if "no_resend" is something someone else could `.on`, then we should use something like https://github.com/sindresorhus/emittery#isnt-eventemitter-synchronous-for-a-reason instead (and ofc then we can't write try catch for it, instead `emit` returns a Promise that can be `.catch`ed)

Reply:
Should I make a ticket about switch Node EventEmitter for emittery (or similar async thing)? It could be prioritized low or whatever, but I think it's a consideration for a library.

Reply:
"no_resend" is meant for internal use only. But anyway, it seems to be another topic so I'll merge this fix if that's ok. Sure you can make a ticket, but maybe someone with more JS experience than me should give his opinion on this.